### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#451] [Esami & Esoneri M-Z]
## Traccia

<img width="492" alt="image" src="https://github.com/user-attachments/assets/a1433176-1c0c-4e6e-a999-2689b4d55c9d">

**Comment by rimaout** [comment]
<img width="600" alt="image" src="https://github.com/user-attachments/assets/0a4e235c-bf28-4264-bb20-61443f918364">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/75faca3d-a803-4374-bfbb-cd87283912a7)
